.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Apollo
+
+on:
+  push:
+    tags: 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
```

Comment:
Need to add a job to deploy to demo server. The "Deploy to demo staging server" job from `push.yml` should be split into a separate shared workflow file and then called from both `push.yml` and here.